minutes/2025-05-02-psc-meeting.md

```diff
@@ -0,0 +1,71 @@
+---
+minutes-recorded-by: "@gadomski"
+---
+
+# 2025-05-02 STAC PSC meeting
+
+In-person in Snowbird, UT after the [2025 CNG Conference](https://conference.cloudnativegeo.org/CNGConference2025).
+In attendance:
+
+- @PowerChell
+- @matthewhanson
+- @m-mohr
+- @jsignell
+- @gadomski
+- @alexgleith
+
+Not in attendance:
+
+- @emmanuelmathot
+
+## Agenda
+
+Agenda was decided during the first part of the meeting via suggestions then triage, in the interest of time.
+
+- Welcome Alex!
+- PSC purpose
+- Communications
+- Community meetings
+- STAC as a OGC community standard
+- Sprint(s)
+
+## Minutes
+
+### Funding and PSC purpose
+
+- Discussed STAC PSC's relationship with the [Cloud-Native Geospatial (CNG) Forum](https://cloudnativegeo.org/), and whether CNG should work like [NumFocus](https://numfocus.org/) and have "sponsored projects" (e.g. [numpy](https://numfocus.org/project/numpy)).
+  (note: NumFocus provides funding and resources as well)
+- Decided on five overarching goals for the PSC:
+  - Interface with CNG, Radiant Earth, and any other organizations and standards bodies (e.g. [OGC](https://www.ogc.org/))
+  - Create and publish a roadmap for the STAC spec, the STAC API spec, and the STAC ecosystem
+  - Filter and publicly identify issues and problems in the STAC ecosystem, and moderate resolutions to those problems when necessary
+  - Identify and solve core specification issues
+  - Provide and participate in forums for the community to discuss and work with STAC
+
+### Communications
+
+- Accepted motion (@gadomski, second @matthewhanson) move "permanent, threaded discussions" to the [CNG Discourse](https://cng.discourse.group) and disable Github discussions for [stac-spec](https://github.com/radiantearth/stac-spec/discussions) and [stac-api-spec](https://github.com/radiantearth/stac-api-spec/discussions)
```

Comment:
Note: The motion, second thing is not really foreseen in the PSC guidelines. We should go back to normal votes next time.

Reply:
I'm fine w/ that ... I just defaulted to a poor imitation of [Robert's Rules](https://en.wikipedia.org/wiki/Robert%27s_Rules_of_Order) out of instinct.